lib/feature/home_page/widgets/filter_bottom_sheet_vm.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:pokedex_async_redux/feature/home_page/widgets/filter_bottom_sheet_connector.dart';
+import 'package:pokedex_async_redux/state/actions/home_page_actions.dart';
+import 'package:pokedex_async_redux/state/app_state.dart';
+
+class FilterBottomSheetVmFactory extends VmFactory<AppState, FilterBottomSheetConnector> {
+  @override
+  Vm fromStore() {
+    return FilterBottomSheetVm(onFilterTap: _onFilterTap);
+  }
+
+  void _onFilterTap(String filterType) => dispatch(GetFilterList(filterKey: filterType));
```

Comment:
do your action segmentation here, instead of doing it inside the `GetFilterList`